config.md

```diff
@@ -12,7 +12,7 @@ This specification uses the following terms:
 ### [Layer](layer.md)
 
 Image filesystems are composed of *layers*.
-Each layer represents a set of filesystem changes in a tar-based [layer format](layer.md), recording files to be added, changed, or deleted relative to its parent layer.
+Each layer represents a set of filesystem changes in a compressed tar-based [layer format](layer.md), recording files to be added, changed, or deleted relative to its parent layer.
```

Comment:
This is currently covered [here][1], and will no longer be true if/when #388 lands.  In any event, compression seems like a property of the later media type, and not a fundamental property of “being a layer”.

[1]: https://github.com/opencontainers/image-spec/blame/d1c7054fae33dd186c78fb126d5c613b233aa5eb/layer.md#L11